go-controller/pkg/config/config.go

```diff
@@ -71,6 +71,9 @@ var (
 
 	// NbctlDaemon enables ovn-nbctl to run in daemon mode
 	NbctlDaemonMode bool
+
+	// EmptyLbEvents enables sending a k8s NeedPods event when packets arrive for a load balancer VIP with no backends configured.
+	EmptyLbEvents bool
```

Comment:
@putnopvut we'll want this key to be in the config file too, so move this to `KubernetesConfig` with the appropriate `gcfg` tag. Since this is inherently a Kubernetes specific feature (for now) I think it makes more sense there.

Reply:
I may have misinterpreted this comment. I added EmptyLbEvents to KubernetesConfig in addition to having it where it is now. This was based on your quote of having this key in "the config file, *too*" . If this should *only* be in the KubernetesConfig, please let me know.

Reply:
@putnopvut so we actually only need this in config.Kubernetes.OVNEmptyLbEvents. The standalone EmptyLbEvents can be removed, because the code will actually take all of [CLI, config file, defaults] and collapse them all into config.Kubernetes.OVNEmptyLbEvents with some priority ordering. Then all the code everywhere can simply refer to config.Kubernetes.OVNEmptyLbEvents and have consisten CLI/config/etc option handling.